src/main/scala/eisner/EisnerPlugin.scala

```diff
@@ -13,21 +13,31 @@ import scala.concurrent.ExecutionContext.Implicits.global
 
 object EisnerPlugin extends AutoPlugin with ReflectionSupport with SnippetSupport {
   object autoImport {
-    val eisnerTopologies = settingKey[Seq[String]]("The fully qualified names of classes implementing org.apache.kafka.streams.Topology")
+    val eisnerColorSink        = settingKey[String]("The color used to represent sinks, see https://www.graphviz.org/doc/info/colors.html")
+    val eisnerColorSubtopology = settingKey[String]("The color used to frame subtopologies, see https://www.graphviz.org/doc/info/colors.html")
+    val eisnerColorTopic       = settingKey[String]("The color used to represent topics, see https://www.graphviz.org/doc/info/colors.html")
+    val eisnerRoughSVG         = settingKey[Boolean]("The flag that controls whether to create SVG using pseudo hand-drawing")
+    val eisnerTargetDirectory  = settingKey[File]("The directory where to store the generated topologies")
+    val eisnerTopologies       = settingKey[Seq[String]]("The fully qualified names of classes implementing org.apache.kafka.streams.Topology")
     val eisnerTopologiesSnippet =
       settingKey[Option[String]]("A scala snippet (including all imports) that evaluates to a Seq[(String, org.apache.kafka.streams.Topology)]")
-    val eisner = taskKey[Seq[File]]("Generates one SVG for each org.apache.kafka.streams.Topology")
+    val eisner = taskKey[Set[File]]("Generates one SVG for each org.apache.kafka.streams.Topology")
   }
 
   import autoImport._
 
   override final def projectSettings: Seq[Setting[_]] = Seq(
+    eisnerColorSink := "black",
+    eisnerColorSubtopology := "lightgrey",
+    eisnerColorTopic := "black",
+    eisnerRoughSVG := true,
+    eisnerTargetDirectory := (Compile / target).value / "eisner",
     eisnerTopologies := Seq.empty,
     eisnerTopologiesSnippet := None,
     eisner := generate.dependsOn(Compile / compile).value
   )
 
-  private[this] final def generate: Def.Initialize[Task[Seq[File]]] = Def.taskDyn {
+  private[this] final def generate: Def.Initialize[Task[Set[File]]] = Def.taskDyn {
```

Comment:
the task itself does not _need_ to return anything, it's not being attached to any [`sourceGenerators`](https://www.scala-sbt.org/1.x/docs/Howto-Generating-Files.html) or such but... I had to switch to `Set` since `FileFunction.cached` expects to return a `Set[File]` (and - tbh - it's also correct to treat these as sets)